scripts/servers_test.py

```diff
@@ -926,7 +926,8 @@ def test_managed_webdriverio_with_invalid_sharding_instances(self) -> None:
     def test_managed_webdriverio(self) -> None:
         popen_calls = self.exit_stack.enter_context(self.swap_popen())
 
-        self.exit_stack.enter_context(servers.managed_webdriverio_server())
+        self.exit_stack.enter_context(
+            servers.managed_webdriverio_server(chrome_version='104.0.5112.79'))
```

Comment:
Can you add some kind of assertion to make sure that this chrome version is used correctly?

Reply:
Done